site/profile/manifests/cvmfs.pp

```diff
@@ -118,4 +119,34 @@
   # 'use_fusefs_home_dirs' policy fix that issue.
   selinux::boolean { 'use_fusefs_home_dirs': }
 
+  # Creates the resource if it does not already exist
+  ensure_resources(profile::cvmfs::local_cvmfs_user, $cvmfs_user)
+}
+
+# Create a local cvmfs user
+class profile::cvmfs::local_cvmfs_user (
+  String $selinux_user = 'unconfined_u',
+  String $mls_range = 's0-s0:c0.c1023',
+  String $cvmfs_user = 'cvmfs-user'
+)
+{
+  user { $cvmfs_user:
+    ensure     => present,
+    forcelocal => true,
+    gid        => '8000131',
```

Comment:
`gid        => '8000131'` is ok, but group should be 

`groups     => 'cvmfs-reserved',`

Reply:
done